### ISSUE TITLE
Show last modification date for posts (and not just creation date)

### ISSUE THREAD

**Issue by kuleszdl** [AngeloStavrow/indigo#92]
It would be great if the post would show both when the post was created as well as when the post was last udpated.

**Comment by AngeloStavrow** [comment]
Hey @kuleszdl, that's an interesting idea! Do you have thoughts on what it could look like, on desktop and on mobile?

**Comment by kuleszdl** [comment]
Yes @AngeloStavrow I thought about just having a second line below (or even better above) the line with the date - maybe also in a different color? On the main page in desktop it maybe even could fit in one line.

Also, I am not sure if it makes sense to have the exact time and time zone on the main page. Probably something like this would fit:

     Posted on June 21st, 2020 and last updated on July 5th, 2020

And in the detailed view:

    Posted on June 21st, 2020 at 21:30 UTC by Daniel Kulesz
    Last updated on July 5th, 2020 at 21:30 UTC by Daniel Kulesz
